pkg/config/config.go

```diff
@@ -25,6 +25,8 @@ type Config struct {
 	Keyring                *KeyringConfig `toml:",omitempty"`
 	Ordering               string
 	ExportCredentialSuffix string
+	// This is used to generate reference URls
+	GrantedApprovalsUrl string
```

Comment:
nitpick: URL is an acronym and acronyms should be capitalised in Go

```suggestion
	GrantedApprovalsURL string
```